multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -163,7 +163,9 @@ write_files:
   - path: /opt/bin/kubectl
```

Comment:
Can we please move this out of PATH (`/tmp`?) or just use this functionality directly when creating the addons.

What this will lead to:

```
$ ls manifest.yaml
manifest.yaml
$ kubectl create -f manifest.yaml
error: open manifest.yaml: no such file or directory
```

I agree this functionality will be nice to have -- but it needs more thought and is not strictly required for this PR. Let's have this be a follow up.